src/oocss/whitespace/_whitespace.scss

```diff
@@ -85,9 +85,8 @@ $whitespace-keys: (p padding, m margin, a '', v -top -bottom, h -left -right, t
  * e.g. @include whitespace((), 4);
  *
```

Comment:
To keep up with the doc format, since you moved the types to a param, can you add that to the comment block?